omd/packages/omd/omdlib/utils.py

```diff
@@ -30,7 +30,11 @@
 
 
 def is_dockerized() -> bool:
-    return os.path.exists("/.dockerenv") or os.path.exists("/run/.containerenv")
+    return (
+        os.path.exists("/.dockerenv")
+        or os.path.exists("/run/.containerenv")
+        or os.environ.get("CMK_CONTAINERIZED") is not None
```

Comment:
I would rather check if the value is "yes".

Reply:
I updated the code accordingly 👍